app/src/pages/trace/SpanFeedback.tsx

```diff
@@ -128,14 +132,20 @@ export function SpanFeedback({ span }: { span: SpanFeedback_annotations$key }) {
   }, [data.spanAnnotations]);
   const hasAnnotations = data.spanAnnotations.length > 0;
   return hasAnnotations ? (
-    <Accordion>
-      <AccordionItem id={"evaluations"} title={"Evaluations"}>
-        <SpanAnnotationsTable annotations={llmAnnotations} />
-      </AccordionItem>
-      <AccordionItem id={"human"} title={"Human Annotations"}>
-        <SpanAnnotationsTable annotations={humanAnnotations} />
-      </AccordionItem>
-    </Accordion>
+    <DisclosureGroup defaultExpandedKeys={["evaluations", "human"]}>
+      <Disclosure id="evaluations">
+        <DisclosureTrigger>Evaluations</DisclosureTrigger>
+        <DisclosurePanel>
+          <SpanAnnotationsTable annotations={llmAnnotations} />
+        </DisclosurePanel>
+      </Disclosure>
+      <Disclosure id="human">
+        <DisclosureTrigger>Human Annotations</DisclosureTrigger>
```

Comment:
same here - I think there should be a separation of the header from trigger and content on the second 
<img width="809" alt="Screenshot 2025-01-02 at 7 51 47 PM" src="https://github.com/user-attachments/assets/457bbb11-20fe-4be0-af6e-8b47af7896be" />

Reply:
Fixed